static/app/views/issueList/customViewsHeader.tsx

```diff
@@ -304,19 +305,21 @@ function CustomViewsIssueListHeaderTabsContent({
               tab.querySort === view.querySort &&
               tab.label === view.name
             ) {
+              if (tab.id === currentViewId) {
+                navigate(
+                  normalizeUrl({
+                    ...location,
+                    query: {
+                      ...queryParamsWithPageFilters,
+                      viewId: view.id,
+                    },
+                  }),
+                  {replace: true}
+                );
+              }
```

Comment:
Had to refactor this useEffect to only replace the viewId in the search params for the view that the user is currently on.  There was no other way to bulk add tabs before, so this was never an issue.